sdk/keyvault/azure-keyvault-certificates/samples/list_operations.py

```diff
@@ -49,11 +49,11 @@
     storage_certificate_poller = client.create_certificate(name=storage_cert_name)
 
     # await the creation of the bank and storage certificate
-    bank_certificate_poller.wait()
-    storage_certificate_poller.wait()
+    bank_certificate = bank_certificate_poller.result()
+    storage_certificate = storage_certificate_poller.wait()
```

Comment:
This implies `result` and `wait` do the same thing.

Reply:
thanks, I accidentally left the wait in.